.github/workflows/zap_build_mapzap.yml

```diff
@@ -1,4 +1,4 @@
-name: Build MapZAP
+name: ZAP - MapZAP (Placeholder)
```

Comment:
this workflow should've been labeled as a placeholder already. it lives in bigquery and the exploratory builds of it were first run manually via sequential queries and then via dbt on the command line